backend/canisters/local_group_index/impl/src/lifecycle/post_upgrade.rs

```diff
@@ -34,6 +40,8 @@ fn post_upgrade(args: Args) {
         utils::cycles::check_cycles_balance(user_index_canister_id)
```

Comment:
This needs to be changed... we no longer use `check_cycles_balance` here and also it was using the user_index... not sure how we both missed that!